CoreFoundation/CMakeLists.txt

```diff
@@ -373,7 +377,8 @@ add_framework(CoreFoundation
                 URL.subproj/CFURLAccess.c
                 URL.subproj/CFURL.c
                 URL.subproj/CFURLComponents.c
-                URL.subproj/CFURLComponents_URIParser.c)
+                URL.subproj/CFURLComponents_URIParser.c
+                ${CF_SWIFT_SOURCES})
```

Comment:
Please use `target_sources` for adding the additional source file.

Reply:
Done.